python/cudf/dataframe/multiindex.py

```diff
@@ -367,6 +367,33 @@ def to_frame(self, index=True, name=None):
             df.columns = name
         return df
 
+    def get_level_values(self, level):
```

Comment:
Did an offline walkthrough of this with @shwina and agreed that this should be reworked to use the MultiIndex `_source_data` instead of `codes`/`levels`